### PR TITLE
Arian proposal

### PR DESCRIPTION
**Pull request by ArianBLC**
Handel + hotStuff proposal

### CODE REVIEW

**Comment by meling** [comment]
LGTM!

File: 2022/Handel signature aggregation in hotStuff.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Recent works at UiS have implemented the HotStuff protocol in Golang. The current implementation supports BLS signatures and also ECDSA signatures. The implementation can be found in [here](https://github.com/relab/hotstuff/). The main task of this project is to implement new signature aggregation schemes such as Handel in the current implementation for HotStuff and evaluate the impact of these mechanisms on the security and performance of the system. 
+
+## Tasks / Milestones
+
+ - Study HotStuff paper carefully and understand the parts of its implementation that are needed for this project, e.g., [the crypto module](https://github.com/relab/hotstuff/tree/master/crypto) could be a starting point.
+ - Study different signature aggregation mechanisms such as Handel, and learn how to work with their libraries in Golang.
+ - Implement some of the signature aggregations in the HotStuff.
```
Comment (meling) [inline 2022/Handel signature aggregation in hotStuff.md]: Minor: remove `the` before `Hotstuff`

Comment (meling) [reply 2022/Handel signature aggregation in hotStuff.md]: Maybe while you are at it: add a bullet item about evaluation

**Comment by leandernikolaus** [comment]
Looks good. 
But maybe add some prerequisites, like: Students should be experienced coding in go and preferably take the blockchain technology course concurrently.

Also, if Hein would agree to it, he could also do the responsible professor part. 
He is the local expert on go programming.

But if you prefer to keep me there, I can do that as well.

File: 2022/Handel signature aggregation in hotStuff.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Implementing Different signature aggregations in HotStuff
+
+## Administrative 
+
+- Supervisor: Arian Baloochestani
+- Responsible Professor: Leander Jehl
+
+## Prerequisites
+
+ - DAT650 Blockchains
```
Comment (leandernikolaus) [inline 2022/Handel signature aggregation in hotStuff.md]: There will probably not be students that already have taken the subject. 
You can write that they should have taken DAT530Distributed systems and that they should be good go programmers.
You can write that it is recommended they do take the blockchain course concurrently.

**Review by leandernikolaus — APPROVED** [review]

**Comment by meling** [comment]
Feel free to add me as the responsible professor, to offload Leander.

**Review by quickfeed-uis — APPROVED** [review]

**Pull request merged**